client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/SecurityDocumentationIT.java

```diff
@@ -109,6 +111,17 @@
 
 public class SecurityDocumentationIT extends ESRestHighLevelClientTestCase {
 
+    public void testGetUsers() throws Exception {
+        RestHighLevelClient client = highLevelClient();
+        addUser(client, "testUser", "testPassword");
+
+        {
+            GetUsersRequest getUsersRequest = new GetUsersRequest();
+            GetUsersResponse getUsersResponse = client.security().getUsers(getUsersRequest, RequestOptions.DEFAULT);
+            assertNotNull(getUsersResponse.getUsers());
+        }
+    }
+
```

Comment:
this is not complete yet, I would simply use `testGetRoles` as a blueprint.
The same with the documentation, use `./docs/java-rest/high-level/security/get-roles.asciidoc` as an example.
And don't forget to add it to `./x-pack/docs/en/rest-api/security.asciidoc`.

Reply:
:+1: